### PR TITLE
Dx 619 Dx 616

### PR DESCRIPTION
**Pull request by jmulford-bw**
## For the Committer

⚠️ Ensure that for this repo (**bandwidth.github.io**) that the pull request change is opened against the branch `gitbook` 

- [ ] I made sure that the site looks great and functions perfectly live
- [ ] I ran splell check
- [ ] I'm ready for these changes to be made live

### General Review

- [ ] All changes/updates requested were intentionally changed or added (no extraneous file or partial updates)
- [ ] The live rendered page behaves and looks like intended.
- [ ] All links work as intended. (click every link to make sure that it takes the user to the expected place).
- [ ] The markdown was rendered to HTML as intended. (Tables look right, code samples are gated properly).
- [ ] Any style or CSS changes are tested on multiple different pages to ensure nothing unexpected broke or changed.
- [ ] I proof-read the live site (`gitbook serve`) and there are no oblivious splelling or grammer misteaks.

### Gitbook Specific

- [ ] Any new documentation pages have been added to the `SUMMARY.md` file so they are rendered to HTML.
- [ ] Any links to other pages within this documentation use relative links (`[read more about ...](guides/voice/voicemail.md)`).
- [ ] Gitbook can install and build the documentation (`gitbook install` & `gitbook build`)

### Code Specific

- [ ] All code has been tested against the live API.
- [ ] All code can be run without throwing errors or **new** warnings.
- [ ] All code is formatted correctly and consistently (spaces, tabs).
- [ ] All code is legible and idiomatic to the language.
- [ ] All code uses sensical method, function, and variable names.
- [ ] All code samples use the appropriate syntax highlighting.
- [ ] Any examples re-use same phone numbers, uuids, variable names, etc... throughout (IE don't have something like `{from: '+19191231234', to: '+19191231234'}` unless it's meant to show the same phone number calling/texting itself ).

## TL;DR

- [ ] I did these things
- [ ] I'm ready for these changes to be made live

### CODE REVIEW

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-DX-619.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-DX-619.s3-website-us-east-1.amazonaws.com/

File: SUMMARY.md [hunk 1]
```diff
@@ -184,7 +184,8 @@
 * [C#](sdks/csharp.md)
 * [Node](sdks/node.md)
 * [Java - _coming soon_](sdks/java.md)
-* [PHP - _coming soon_](sdks/php.md)
+* [Node](sdks/node.md)
```
Comment (ghost) [inline SUMMARY.md]: added node twice

File: voice/bxml/verbs/speakSentence.md [hunk 2]
```diff
@@ -253,7 +266,7 @@ Console.WriteLine(response.ToBXML());
 response = Bandwidth::Voice::Response.new()
 speak_sentence = Bandwidth::Voice::SpeakSentence.new({
     :sentence => 'Hello, you have reached the home of <lang xml:lang="es-MX">Antonio Mendoza</lang>.Please leave a message.',
-    :voice => "Sophia"
+    :voice => "sophia"
```
Comment (ghost) [inline voice/bxml/verbs/speakSentence.md:269]: are our voice case sensitive? If so, let's handle that in the SDK with a `.lowerCase()` or w/e for each of these sensitivities

**Review by ghost — CHANGES_REQUESTED** [review]
Few small changes

File: SUMMARY.md (hunk 1 above)
Comment (jmulford-bw) [reply SUMMARY.md]: Oops. Fixed 👍

File: voice/bxml/verbs/speakSentence.md (hunk 2 above)
Comment (jmulford-bw) [reply voice/bxml/verbs/speakSentence.md:269]: It is. I copy-pasted an example that has it uppercase and it failed the BXML validation. I'm not against going that route, but if we do I'd like to confirm with the voice team that all voices will be lowercase

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-DX-619.s3-website-us-east-1.amazonaws.com/

**Review by ghost — APPROVED** [review]

**Review by jchavez443 — APPROVED** [review]

**Pull request merged**